executable/AndroidManifest.ksy

```diff
@@ -0,0 +1,187 @@
+meta:
+  id: android_manifest
+  title: Android Binary manifest xml
+  file-extension: xml
+  endian: le
+doc: |
+    Android manifest has its own binary xml format.
+    It is not documented.
+seq:
+  - id: hdr
+    type: header
+  - id: strings_table_info_size
+    size: 4
+    doc: |
+      Size of all fields related to strings table.
+      Size of strings table info often counts from 0x8 byte until
+      first byte of strings table.
+      It should be equal 7*4 + 4*strings_count. 
+      7 - is a number of previous fields (4 bytes size).
+      If it is not equal to that then there is an error in manifest strings offset 
+  - id: styles_offset
+    size: 4
+  - id: strings_offset_table
+    size: hdr.strings_count*4
+    doc: table of strings offsets
+  - id: strings
+    type: manifest_string
+    repeat: expr
+    repeat-expr: hdr.strings_count*2
+    doc: |
+      Table of all strings in manifest. Strings are in UTF-16 encoding.
+      Each string has its length in the beginning.
+  - id: padding
+    size: (4 - _io.pos) % 4
+    doc: | 
+      Resource header should be aligned to 4 bytes.
+      Kaitai doesn't have 'align' keyword so we use this hack.
```

Comment:
Depends on https://github.com/kaitai-io/kaitai_struct/issues/12 — it's better to note missing functionality with pointers to where that work is tracked. Please don't call "Kaitai Struct" as "Kaitai".

```suggestion
  - id: padding
    size: (4 - _io.pos) % 4
    doc: | 
      Resource header should be aligned to 4 bytes.
      Workaround for https://github.com/kaitai-io/kaitai_struct/issues/12
```